pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -487,6 +507,8 @@ class Build {
                                             context.string(name: 'VENDOR_TEST_REPOS', value: VENDOR_TEST_REPOS),
                                             context.string(name: 'VENDOR_TEST_BRANCHES', value: VENDOR_TEST_BRANCHES),
                                             context.string(name: 'VENDOR_TEST_DIRS', value: VENDOR_TEST_DIRS)],
+                                            context.string(name: 'RELATED_NODES', value: relatedNodeLabel)],
```

Comment:
Please remove the extra `]`. Same as line 509

Reply:
Removed. Thanks!